src/components/CategoryNavContainer/CategoryNav/CategoryNav.scss

```diff
@@ -122,8 +122,7 @@
     }
 
     .housing {
-      // 600 for housing doesn't pass WCAG AA, so we use 800
-      background: $color-housing--800;
+      background: $color-housing--900; // 900 to pass WCAG AA
```

Comment:
Unrelated to the task but I was setting housing to use the 900 colour elsewhere so updated here for consistency